index.html

```diff
@@ -33,23 +33,131 @@ <h1>This is a branch called part 1</h1>
+								</ul>
+					    </div>
+					    <div role="tabpanel" class="tab-pane" id="deserts">
+								<ul>
+									<li class="bold">Chocolate Lava Cake</li>
+										<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Velit quas quod voluptas iure suscipit dolor, dignissimos pariatur! Facere ullam fuga sequi commodi, saepe excepturi nisi!</p>
+									<li class="bold">Strawberry Cheesecake</li>
+										<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Velit quas quod voluptas iure suscipit dolor, dignissimos pariatur! Facere ullam fuga sequi commodi, saepe excepturi nisi!</p>
+									<li class="bold">Creme Brulee</li>
+										<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Velit quas quod voluptas iure suscipit dolor, dignissimos pariatur! Facere ullam fuga sequi commodi, saepe excepturi nisi!</p>
+									<li class="bold">Fruit Tarte</li>
+										<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Velit quas quod voluptas iure suscipit dolor, dignissimos pariatur! Facere ullam fuga sequi commodi, saepe excepturi nisi!</p>
+								</ul>
+					    </div>
+					    <div role="tabpanel" class="tab-pane" id="cocktails">
+								<ul>
+									<li class="bold">Lemon Drop</li>
+										<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Velit quas quod voluptas iure suscipit dolor, dignissimos pariatur! Facere ullam fuga sequi commodi, saepe excepturi nisi!</p>
+									<li class="bold">Mai Tai</li>
+										<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Velit quas quod voluptas iure suscipit dolor, dignissimos pariatur! Facere ullam fuga sequi commodi, saepe excepturi nisi!</p>
+									<li class="bold">Gin &amp; Tonic</li>
+										<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Velit quas quod voluptas iure suscipit dolor, dignissimos pariatur! Facere ullam fuga sequi commodi, saepe excepturi nisi!</p>
+									<li class="bold">House Special</li>
+										<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Velit quas quod voluptas iure suscipit dolor, dignissimos pariatur! Facere ullam fuga sequi commodi, saepe excepturi nisi!</p>
+								</ul>
+					    </div>
+					  </div>
+					</div>
+				</div>
+			</div>
+		</section>
+
+
+
+		<section class="container-fluid footer">
+			<div class="row">
+				<div class="col-xs-12 col-sm-12 col-md-6 col-md-push-2">
+					<h2>Make A Reservation</h2>
+					<ul>
+						<li class="bold">Chefs Table</li>
+						<li>123 Market Street, Suite 900</li>
+						<li>Fresno, CA 95432</li>
+						<li>P: (559) 456-7890</li>
+						<i class="fab fa-facebook-square"></i>
+						<i class="fab fa-twitter"></i>
+						<i class="fab fa-instagram"></i>
+					</ul>
+				</div>
+				<div class="col-xs-12 col-sm-12 col-md-6 col-md-pull-1">
```

Comment:
col-md-pull-1

**should be matching the push you did at line 144 col-md-push-2**
col-md-pull-2